tpgtools/overrides/orgpolicy/beta/policy.yaml

```diff
@@ -0,0 +1,10 @@
+- type: CUSTOM_IMPORT_FUNCTION
+  details:
+    function: resourceOrgPolicyPolicyCustomImport
+- type: USE_DCL_ID
+- type: ENUM_BOOL
```

Comment:
Non-review question: how would contributors be able to to recognize that a boolean should have this override applied to it?

Reply:
If the field is a boolean in the DCL and it's necessary to be able to distinguish between `true`, `false`, and `nil`, then it needs to be an enum/string in terraform which can be `"TRUE"`, `"FALSE"`, or `""`.
In this case the distinction is important because these fields conflict with each other and setting `enforce` to `false` is different than leaving all of them unset.

Reply:
If it needs to have an unset case which is distinct from false.  This would be a good place to insist we write those override docs!

Reply:
Yes if we could add a one-pager doc per override added, that would help. I don't quite remember if we had a specific place where we've kept them all, @ndmckinley do you remember? Maybe we can chat in an internal chat about it.